app/extensions.py

```diff
@@ -29,6 +28,7 @@ def __init__(self, uid: str, user_info: dict):
         self._raw_dict['uid'] = uid
 
     def get_dict(self) -> dict:
+        print (self._raw_dict)
```

Comment:
@alejandrogznz Nit: we should get rid of the print?

Reply:
Yup I can change it right now.